specs/MultiProfile.md

```diff
@@ -363,16 +394,42 @@ interface ICoreWebView2Profile2 : ICoreWebView2Profile {
   [propget] HRESULT CookieManager([out, retval] ICoreWebView2CookieManager** cookieManager);
 }
 
-[uuid(1c1ae2cc-d5c2-ffe3-d3e7-7857035d23b7), object, pointer_default(unique)]
-interface ICoreWebView2Profile3 : ICoreWebView2Profile2 {
-  /// All webviews on this profile will be closed, and the profile will be marked for deletion.
-  /// After the Delete() call completes, The render process of webviews on this profile will
-  /// asynchronously exit with the reason:`COREWEBVIEW2_PROCESS_FAILED_REASON_PROFILE_DELETED`.
-  /// See 'COREWEBVIEW2_PROCESS_FAILED_REASON::COREWEBVIEW2_PROCESS_FAILED_REASON_PROFILE_DELETED'
-  /// for more details. The profile directory on disk will be actually deleted when the browser
-  /// process exits. Webview2 creation will fail with the HRESULT is ERROR_INVALID_STATE(0x8007139FL)
-  /// if you create it with the same name as a profile that is being deleted.
-  HRESULT Delete(); 
+[uuid(2765B8BD-7C57-4B76-B8CC-1EC940FF92CC), object, pointer_default(unique)]
+interface ICoreWebView2StagingProfile7 : IUnknown {
+  /// After the API is called, the profile will be marked for deletion. The
```

Comment:
Is the profile considered marked for deletion when this API returns?  Or does it really mean an action to mark it for deletion has been queued up?  If the profile is marked for deletion during function execution, Wordsmithing:

> This API marks a profile for deletion.

Reply:
The API will send an async MOJOM message to the browser to mark it for deletion.  When this API returns, the MOJOM message may have not completed. So, it really means an action to mark it for deletion has been queued up.